chromadb/utils/embedding_functions.py

```diff
@@ -479,6 +479,59 @@ def __call__(self, texts: Documents) -> Embeddings:
         return embeddings
 
 
+class VoyageAIEmbeddingFunction(EmbeddingFunction):
+    def __init__(self, api_key: str, model_name: str = "voyage-01", batch_size: int = 8):
+        """
+        Initialize the VoyageAIEmbeddingFunction.
+
+        Args:
+        api_key (str): Your API key for the HuggingFace API.
+        model_name (str, optional): The name of the model to use for text embeddings. Defaults to "voyage-01".
+        batch_size (int, optional): The number of documents to send at a time. Defaults to 8 (The max supported 3rd Nov 2023).
+        """
+        if batch_size > 8:
+            print(f"Voyage AI as of (3rd Nov 2023) has a batch size of max 8")
+
+        if not api_key:
+            raise ValueError("Please provide a VoyageAI API key.")
+
+        try:
+            import voyageai
+            from voyageai import get_embeddings
+        except ImportError:
+            raise ValueError("The VoyageAI python package is not installed. Please install it with `pip install voyageai`")
+
+        voyageai.api_key = api_key  # Voyage API Key
+        self.batch_size = batch_size
+        self.model = model_name
+        self.get_embeddings = get_embeddings
+
+    def __call__(self, texts: Documents) -> Embeddings:
```

Comment:
as of 0.4.16 the input parameter here is "input" not "texts".

https://github.com/chroma-core/chroma/blob/933d46d8c239b630205a13481e4bbff047398b11/chromadb/utils/embedding_functions.py#L57

Reply:
Thanks

Reply:
Changed to input in new commit.